### DISCUSSION TITLE
Welcome to Repoistala Discussions!

### DISCUSSION THREAD

**Discussion by Kalina67** [Kalina67/Repoistala#2] [Announcements]
Can someone show an example of JavaScript Object Protection?

**Comment by Aref-abt** [comment]
Hello, Sure here is a simple one:

// Returns true if object is sealed
Object.isSealed(object)

// Prevents any changes to an object
Object.freeze(object)

// Returns true if object is frozen
Object.isFrozen(object)

U might not wanna use this for now, cause you just started but nice question!
